lasy/utils/laser_utils.py

```diff
@@ -682,6 +682,59 @@ def create_grid(array, axes, dim):
     return grid
 
 
+def get_container(dim, grid, omega0, transform=True, n_dump=0, backend="NP"):
+    """
+    Export field to axiprop container, clean, and trasnform to frequency.
+
+    Parameters
+    ----------
+    dim : string
+        Dimensionality of the array. Options are:
+        - 'xyt': The laser pulse is represented on a 3D grid:
+                 Cartesian (x,y) transversely, and temporal (t) longitudinally.
+        - 'rt' : The laser pulse is represented on a 2D grid:
+                 Cylindrical (r) transversely, and temporal (t) longitudinally.
+
+    grid : a Grid object.
+        It contains a ndarrays (V/m) with
+        the value of the envelope field and the associated metadata
+        that defines the points at which the laser is defined.
+
+    omega0 : scalar
+        Angular frequency at which the envelope is defined.
+
+    transform: bool
+        Whether to transform field to the spectral domain.
+
+    n_dump: int
+        Number of cells to be used for attenuating boundaries
+
+    backend : string (optional)
+        Backend used by axiprop (see AVAILABLE_BACKENDS in axiprop
+        documentation for more information).
+    """
+    time_axis_indx = -1
+
+    if dim == "rt":
+        # Make a list of containers for RT geometry
+        Container = []
```

Comment:
I think we capitalise e.g. class names, not lists here, so should this really be capitalised?